engine/hatchery/swarm/swarm.go

```diff
@@ -386,7 +394,7 @@ func (h *HatcherySwarm) SpawnWorker(spawnArgs hatchery.SpawnArguments) (string,
 	}
 
 	//start the worker
-	if err := h.createAndStartContainer(dockerClient, args, spawnArgs); err != nil {
+	if err := h.createAndStartContainer(ctx, dockerClient, args, spawnArgs); err != nil {
```

Comment:
invalid operation: h (variable of type *HatcherySwarm) has no field or method createAndStartContainer